frontend/packages/pipelines-plugin/src/components/catalog/hooks/useTaskResources.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { coFetch } from '@console/internal/co-fetch';
+import { TEKTON_HUB_API_ENDPOINT } from '../const';
+
+const useTaskResources = () => {
```

Comment:
this one `useTaskResources` and `useTaskCategories` look similar can we have one hook and control `categories/resources` via props?

Reply:
Refactored to one Hook and passing the Url as prop.